swagger-spec/identifiers/list.yaml

```diff
@@ -1 +1,80 @@
 # /identifiers/{node_id}/identifiers/
+get:
+  summary: List all identifiers
+  description: >-
+    List of all identifiers for a given node.
+
+    ####Returns
+
+    Returns a JSON object containing `data` and `links` keys.
+
+
+    The `data` key contains an array of identifiers.
+    Each resource in the array is a separate identifier object.
+
+
+    The `links` key contains a dictionary of links that can be used for [pagination](#Introduction_pagination).
+
+    ####Filtering
+
+    You can optionally request that the response only include identifiers that match your filters by utilizing the `filter` query parameter, e.g.
+    https://api.osf.io/v2/nodes/?filter[category]=doi.
```

Comment:
I think the example link should be https://api.osf.io/v2/identifiers/73pnd/identifiers/?filter[category]=doi here.